firefox-ios/firefox-ios-tests/Tests/XCUITests/Selectors/AddCreditCardSelectors.swift

```diff
@@ -0,0 +1,122 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+protocol AddCreditCardSelectorsSet {
+    var TITLE: Selector { get }
+    var NAME_FIELD: Selector { get }
+    var CARD_NUMBER_FIELD: Selector { get }
+    var EXPIRATION_FIELD: Selector { get }
+    var CLOSE_BUTTON: Selector { get }
+    var NAME_FIELD_BUTTON: Selector { get }
+    var CARD_NUMBER_FIELD_BUTTON: Selector { get }
+    var EXPIRATION_FIELD_BUTTON: Selector { get }
+    var SAVE_BUTTON: Selector { get }
+    var INVALID_CARD_NUMBER_LABEL: Selector { get }
+    var INVALID_EXPIRATION_LABEL: Selector { get }
+    var USE_SAVED_CARD_BUTTON: Selector { get }
+    var MANAGE_CARDS_BUTTON: Selector { get }
+    var all: [Selector] { get }
+}
+
+struct AddCreditCardSelectors: AddCreditCardSelectorsSet {
+    private enum IDs {
+        static let addCreditCard = "Add Credit Card"
```

Comment:
Sorry if you've replied in another PR. Do we use the accessibility identifiers from the developers? When the developers update the identifiers over there, we do not have to update these strings here.